packages/vaadin-lumo-styles/gulpfile.js

```diff
@@ -0,0 +1,192 @@
+/* eslint-env node */
+'use strict';
+
+const gulp = require('gulp');
+const sort = require('gulp-sort');
+const iconfont = require('gulp-iconfont');
+const fs = require('fs');
+const svgpath = require('svgpath');
+const svgmin = require('gulp-svgmin');
+
+/**
+ * Normalize file sort order across platforms (OS X vs Linux, maybe others).
+ *
+ * Before: `[..., 'eye-disabled', 'eye', ...]`
+ * After:  `[..., 'eye', 'eye-disabled', ...]`
+ *
+ * Order of appearance impacts assigned Unicode codepoints, and sometimes build diffs.
+ *
+ * @see https://github.com/nfroidure/svgicons2svgfont/pull/82
+ * @see https://github.com/nfroidure/svgicons2svgfont/blob/master/src/filesorter.js
+ * @see http://support.ecisolutions.com/doc-ddms/help/reportsmenu/ascii_sort_order_chart.htm
+ */
+function sortIconFilesNormalized(file1, file2) {
+  return file1.replace(/-/g, '~').localeCompare(file2.replace(/-/g, '~'), 'en-US');
+}
+
+gulp.task('icons', async function () {
+  const folder = 'icons/svg/';
+  let glyphs;
+
+  // Optimize the source files
+  gulp
```

Comment:
Note: it would be nice to use the async/await syntax to make the sequence of Gulp tasks completely clear, e.g:

```js
await new Promise((resolve) => {
  gulp.task('icons')
    .on('finish', resolve)
})

// or

await iconsTask();
```